docs/book/v1/intro.md

```diff
@@ -22,11 +22,11 @@ namespace Zend\Expressive\Authentication;
 interface UserInterface
 {
     /**
-     * Get the username
+     * Get the unique user identity (id, username, email address or ...)
      *
      * @return string
      */
-    public function getUsername(): string;
+    public function getIdentity(): string;
```

Comment:
My understanding of #14 is that the reporter was also requesting that this method could return more than just a string - an object, for instance? If that's the case, we'd also need to remove the return type here, and modify the `@return` annotation to reflect that.

That said, I'd prefer we just have it return a string, and then the individual `UserInterface` implementations could provide other details via additional properties or methods.

Reply:
> Get the unique user identity (id, username, email address or ...)

With id meant an integer or an UUID even. But I guess using `return (string) $this->id;` would solve the return type.

Reply:
@weierophinney and @xtreamwayz ok, so we'll stay with a `string` return.

Reply:
We are using single space before colon in return type declaration (consistency)